src/helpers/settings.php

```diff
@@ -156,14 +156,18 @@ function get_trusted_proxies()
      */
     function get_project_info()
     {
-        $settings = get_directus_settings_by_keys(['project_name', 'logo']);
+        $settings = get_directus_settings_by_keys(['project_name', 'logo','color','project_icon','project_image']);
```

Comment:
@benhaynes  - Here you go! :) I added these 3 variables in '/' endpoint as well as `project_icon` and `project_image` did not exist in the settings so added them in the `directus_settings` table.